src/index.ts

```diff
@@ -33,12 +33,12 @@ const PKG = require('../../package.json');
 const v1 = require('./v1');
 
 import {Snapshot} from './snapshot';
-import {Subscription, SubscriptionMetadata, SubscriptionMetadataRaw} from './subscription';
+import {Subscription, SubscriptionMetadataRaw} from './subscription';
 import {Topic, PublishOptions} from './topic';
 import {CallOptions} from 'google-gax';
 import {Readable} from 'stream';
 import {google} from '../proto/pubsub';
-import {ServiceError} from 'grpc';
+import {ServiceError, ChannelCredentials} from 'grpc';
```

Comment:
I dont think you should use channel credential from grpc instead use credential from google auth

Reply:
at line 572, used function return type is ChannelCredentials which us assigning value to sslCreds, thought better to use the same type.